app/src/lib/branches/types.ts

```diff
@@ -115,9 +109,9 @@ export class CombinedBranch {
 	}
 
 	currentState(): BranchState | undefined {
-		if (this.vbranch) return BranchState.VirtualBranch;
 		if (this.pr) return BranchState.PR;
 		if (this.remoteBranch) return BranchState.RemoteBranch;
+		if (this.vbranch) return BranchState.VirtualBranch;
```

Comment:
@PavelLaptev I've reordered these if statements because it seemed off to me, but I wanted to get your thoughts.

Reordering these means that if a vbranch or remoteBranch has an associated pull request, in the sidebar it will show the PR icon rather than the remoteBrach icon or vbranch icon.

Example:

This is an unapplied vbranch which has a corresponding remote branch and a corresponding  PR; previously it would display the vbranch icon; now it displays the green PR symbol as displayed below:
<img width="660" alt="Screenshot 2024-05-24 at 17 13 13" src="https://github.com/gitbutlerapp/gitbutler/assets/13354155/979a6af6-96a2-420c-9ee6-a11c26687dbe">

Reply:
Yes, it make sense. Thank you 👍